File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/typescript/F19.P1.ts
// BEGIN PROBE F19.P1
async function run() {
  const first = (async () => 20)();
  const second = (async () => 22)();
  const sum = await first + await second;
  return sum;
}
// END PROBE F19.P1

run().then(s => { console.log(s); });

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/python/F19.P1.py
# BEGIN PROBE F19.P1
from concurrent.futures import ThreadPoolExecutor

with ThreadPoolExecutor() as pool:
    first = pool.submit(lambda: 20)
    second = pool.submit(lambda: 22)

sum = first.result() + second.result()
# END PROBE F19.P1


print(sum)
